locales/base/translation.json

```diff
@@ -1689,6 +1689,14 @@
       "title": "Insufficient gas token balance",
       "description": "You don't have enough {{feeCurrencies}} to cover gas fees. Please add {{feeCurrencies}} to your wallet to complete this swap.",
       "dismissButton": "Dismiss"
+    },
+    "transactionDetails": {
+      "networkFee": "{{networkName}} Network Fee",
+      "swapFee": "{{appName}} Fee",
+      "swapFeeWaived": "Free",
+      "slippagePercentage": "Slippage Tolerance",
+      "networkFeeInfo": "You are swapping on the {{networkName}} Network.\n\nThis estimated network fee is required to conduct a transaction on the {{networkName}} Network. This fee secures your transaction and is paid to network validators.",
```

Comment:
> paid to network validators

Does this claim holds true given that on Celo 80% of base fee is going to a [Carbon Offset Fund](https://blog.celo.org/ultragreen-money-c677e7508abb)?

And on Ethereum the base fee is simply burned.

I wonder if carbon offestting thing can be appealing to some people using Celo (e.g. in the Netherlands some sellers tend to emphasize their environmental impact in their product marketing)?

That having said, I think the most neutral phrasing could probably be like "This fee secures your transaction and is keeps the network operational".

Reply:
> Does this claim holds true given that on Celo 80% of base fee is going to a [Carbon Offset Fund](https://blog.celo.org/ultragreen-money-c677e7508abb)?

Small nit: 80% is burned, 20% goes to the Carbon Offset Fund.

<img width="701" alt="Screenshot 2023-11-03 at 11 26 51" src="https://github.com/valora-inc/wallet/assets/57791/c1210808-8fd2-48e4-9d99-6a3e983945e3">

Reply:
TY for picking this up, i copied what was in designs without thinking about it. I modified the copy to your suggestion @bakoushin - i'll check with Nitya and Laura if we want chain specific messaging. if that's the case i'll follow up with a separate PR